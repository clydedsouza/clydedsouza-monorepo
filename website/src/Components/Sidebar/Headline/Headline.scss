@import '../../../Sass/variables';

.headline {
  font-family: $heading-font;

  h1 {
    color: white;
    -webkit-text-fill-color: #fff;
    margin: 4px 0;
    font-weight: bold;
    font-family: $heading-font;

    @include for-tablets-down {
      margin: 21px 0 9px 0;
    }

    @include for-large-desktop-up {
      font-size: 46px;
    }
  }

  div {
    margin: 2vh 0px;

    @include for-tablets-down {
      margin: 4px 0;
    }

    p {
      font-size: 19px;
      width: 100%;
      margin: 9px 0;

      @include for-large-desktop-up {
        font-size: 24px;
      }

      small {
        padding: 9px 0;
      }
    }
  }
}
