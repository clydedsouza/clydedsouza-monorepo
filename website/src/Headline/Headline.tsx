function Headline() {
  return (
    <div>
      <div id="sidebar">
        <h1>Clyde DSouza</h1>
        <div>
          <p>Software Engineer and Author</p>
          <p>
            <small>Auckland, New Zealand</small>
          </p>
        </div>
        <div>
          <span>
            <a
              href="https://linkedin.com/in/clydedz/"
              target="_blank"
              rel="noreferrer"
            >
              <i className="fab fa-linkedin"></i>
            </a>
          </span>
          <span>
            <a
              href="mailto:clyde@clydedsouza.net?Subject=Hi%20Clyde%21&Body=Hi%20Clyde,%20%0D%0DI%20wanted%20to%20get%20in%20touch%20with%20you.%0D%0DThanks,%20%0D%0DSent%20from%20https%3A%2F%2Fclydedsouza.net"
              target="_blank"
              rel="noreferrer"
            >
              <i className="fas fa-envelope"></i>
            </a>
          </span>
          <span>
            <a
              href="https://twitter.com/clydedz"
              target="_blank"
              rel="noreferrer"
            >
              <i className="fab fa-twitter"></i>
            </a>
          </span>
          <span>
            <a
              href="https://github.com/ClydeDz/"
              target="_blank"
              rel="noreferrer"
            >
              <i className="fab fa-github"></i>
            </a>
          </span>
          <span>
            <a
              href="https://www.youtube.com/@clydedz"
              target="_blank"
              rel="noreferrer"
            >
              <i className="fab fa-youtube"></i>
            </a>
          </span>
          <span>
            <a
              href="https://medium.com/@clydedz"
              target="_blank"
              rel="noreferrer"
            >
              <i className="fab fa-medium"></i>
            </a>
          </span>
        </div>
        <div>
          <p>Purchase my book Mama, Tell Me a Story from these platforms.</p>
        </div>
      </div>
    </div>
  )
}

export default Headline
