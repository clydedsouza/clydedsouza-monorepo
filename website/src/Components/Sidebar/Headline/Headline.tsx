import './Headline.scss'

function Headline() {
  return (
    <div className="headline">
      <h1>Clyde DSouza</h1>
      <div>
        <p>Software Engineer and Author</p>
        <p>
          <small>Auckland, New Zealand</small>
        </p>
      </div>
    </div>
  )
}

export default Headline
