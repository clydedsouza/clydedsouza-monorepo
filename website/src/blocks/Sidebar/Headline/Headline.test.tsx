import { render } from "@testing-library/react";
import Headline from "./Headline";

describe("Headline", () => {
  it("should render headline and tagline", () => {
    const { container } = render(<Headline />);
    expect(container).toMatchSnapshot();
  });
});
