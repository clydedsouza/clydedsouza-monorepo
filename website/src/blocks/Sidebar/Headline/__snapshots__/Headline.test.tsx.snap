// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline should render headline and tagline 1`] = `
<div>
  <div
    class="headline"
  >
    <h1>
      Clyde D'Souza
    </h1>
    <div>
      <p>
        Software Engineer and Author
      </p>
      <p>
        <small>
          Auckland, New Zealand
        </small>
      </p>
    </div>
  </div>
</div>
`;
